documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -805,6 +805,20 @@ even if the even is generated in a different node to where the listener is
 registered. To make them marshallable, either make them extend `Serializable`,
 `Externalizable`, or provide a custom `Externalizer` for them.
 
+==== Skiping notifications
```

Comment:
Minor typo. "Skiping" > "Skipping". Also titles should use probably use caps. "Skipping Notifications"

Reply:
ok